### ISSUE TITLE
host GNIS-LD vocabulary

### ISSUE THREAD

**Issue by mbjones** [DataONEorg/sem-prov-ontologies#84]
The USGS names vocabulary is GNIS, which has been converted to linked data format as GNIS-LD. See paper:

Regalia B, Janowicz K, Mai G, Varanka D, Usery EL (2018) GNIS-LD: Serving and Visualizing the Geographic Names Information System Gazetteer as Linked Data. The Semantic Web, :528–540. https://doi.org/10.1007/978-3-319-93417-4_34

The endpoints for the vocabulary and sparql service have been published a http://gnis-ld.org/, and we might consider hosting this vocabulary for use across DataONE and other systems that need to annotate data against geographic features.

**Comment by amoeba** [comment]
@mbjones just for clarification, are you thinking we'd re-host gnis-ld.org (it seems dead now) in its own right and also look into integrating it into our existing work?

Wayback got a snapshot of the site last year at http://web.archive.org/web/20200813083037/http://gnis-ld.org/ and Blake has what looks like most of the software + pipeline stuff at https://github.com/blake-regalia. So if we were interested in taking over maintenance of the vocab, I think we have a pretty good head start.

**Comment by mbjones** [comment]
Yeah, the idea that @mpsaloha and I discussed, and that @mpsaloha discussed with Jano, was that we might host gnis-ld.org as part of our graph infrastructure at DataONE. To the extent that it is compatible with our approaches, that could make sense. We wouldn't necessarily have to duplicate everything that Blake did if it makes sense to put some of this in another production triple store, but the fastest way to get it all running may very well be to simply run their infrastructure completely at first. At this point, I think we were looking for feasibility and guidance. I think Mark is going to talk to Jano again before we move further on it.

**Comment by amoeba** [comment]
Sounds great. Looking forward to kicking the tires.
